File: archive/j/julia/baklava.jl
#!/usr/bin/julia

function main()
    for i = 0:10
        print(" "^(10 - i))
        println("*"^(i * 2 + 1))
    end 

    for i = 9:-1:0
        print(" "^(10 - i))
        println("*"^(i * 2 + 1))
    end
end

main()

File: archive/p/perl/baklava.pl
#!/usr/bin/env perl
use v5.42;

my $size = 10;

for my $i ( -$size .. $size ) {
    my $spaces = abs($i);
    my $stars  = 2 * ( $size - $spaces ) + 1;

    say ' ' x $spaces, '*' x $stars;
}
